src/main/java/com/adobe/target/edge/client/http/TargetAsyncIdleConnectionMonitorThread.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.target.edge.client.http;
+
+import java.util.concurrent.TimeUnit;
+
+import com.adobe.target.edge.client.ClientConfig;
+import kong.unirest.apache.AsyncIdleConnectionMonitorThread;
+import org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager;
+
+public class TargetAsyncIdleConnectionMonitorThread extends AsyncIdleConnectionMonitorThread {
+
+  public static final long CONNECTION_CHECK_INTERVAL_MS = 1000;
+  private PoolingNHttpClientConnectionManager connectionManager;
+  private long evictIdleConnectionsAfterSecs = 15;
+
+  public TargetAsyncIdleConnectionMonitorThread(PoolingNHttpClientConnectionManager connMgr) {
```

Comment:
Doesn't look like we're using this constructor.  Delete it?

Reply:
Fixed